hyperon_das/das.py

```diff
@@ -276,6 +304,13 @@ def get_links(
             link_type (str): Link type being searched (can be '*' when link_targets is not None).
             target_types (List[str], optional): Template of target types being searched.
             link_targets (List[str], optional): Template of targets being searched (handles or '*').
+        
+        Keyword Args:   
+            no_iterator (bool, optional): Set to False to return an interator. Defaults to True.
+            cursor (Any, optional): Start point to get the links.
```

Comment:
It's impossible to understand what `cursor` does only by this explanation. I believe the behavior determined by `cursor` and `no_iterator == false` should be better explained.

Reply:
Please fix other occurrences of these parameters in the API